janus_server/src/task.rs

```diff
@@ -303,7 +326,8 @@ struct SerializedTask {
     min_batch_duration: Duration,
     tolerable_clock_skew: Duration,
     collector_hpke_config: SerializedHpkeConfig,
-    agg_auth_tokens: Vec<String>,          // in unpadded base64url
+    aggregator_auth_tokens: Vec<String>, // in unpadded base64url
```

Comment:
Note that this rename will change our task definition file format. (in addition to the new member below) I think we have exactly one task YAML file, so backwards compatibility isn't a concern.

Reply:
Agreed -- for now, backwards-compatibility of the Task serialization format is not a requirement.